spec.bs

```diff
@@ -148,6 +148,10 @@ dictionary URLPatternInit {
   USVString baseURL;
 };
 
+dictionary URLPatternOptions {
+  boolean ignoreCase;
```

Comment:
I think we should make this `= false` here to provide a default.

Reply:
Did you actually mean to add the default in the ctor, instead? I am asking, because I am not sure with the syntax. Thinking from TS perspective, it seems that the `dictionary` gets translated to `interface` and there such default values might be odd. Can you please confirm?

Reply:
Never mind. Looked up the Web IDL specs 😄